README.md

```diff
@@ -1,7 +1,77 @@
 # Grateful Together API
 
+## Abstract: 
+This is the Backend API for the [Grateful Together App](https://www.FIX-THIS.com). Grateful Together is an application that allows groups of people to keep a central, easy to use, daily gratitude journal.
 
-### JSON Contract
+### Preview of App:
+Live Deployment link
```

Comment:
Is this a reminder/placeholder for the link to the FE/interactive view of the app for users?

Reply:
Yes, we'll update it with the deployed site when that is up